README.md

```diff
@@ -167,7 +167,9 @@ Our recommended filesystem layout looks like this:
 
 ## Example
 
-The [example](example) folder contains a complete solution that shows how to:
+**NOTE**: Our recommended example of atmos is found on our documentation site: https://docs.cloudposse.com/tutorials/atmos-getting-started/
```

Comment:
@osterman @aknysh I'm wondering if we should strike this example altogether and just direct folks to the docs site / tutorial... I feel like maintaining both is a bit cumbersome and the tutorial does a decent walkthrough. What do you two think?

Reply:
@Gowiem sounds good. We can maintain the doc in one place

Reply:
The tutorial doesn't cover everything in the example. Setting up/using `vendir` is missing.  Can the docs be updated before the example gets removed?